src/index.html

```diff
@@ -2,17 +2,61 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+
     <title>Product cards</title>
-    <link
-      rel="stylesheet"
-      href="./styles/index.scss"
-    />
+
+    <link rel="stylesheet" href="./styles/index.scss"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com"/>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet"/>
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="product" data-qa="card">
+
+      <div class="product__logo"></div>
+
+      <div class="product__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product__code">
+        Product code: 195434
+      </div>
```

Comment:
use semantic tags for text content
```suggestion
      <h2 class="product__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>

      <p class="product__code">
        Product code: 195434
      </p>
```